samples/templates/default-azuredeploy-docker.json

```diff
@@ -146,7 +146,7 @@
         },
         "sqlSchemaAutomaticUpdatesEnabled": {
             "type": "bool",
-            "defaultValue": "[if(equals(parameters('solutionType'),'FhirServerCosmosDB'), bool('false'), '')]",
+            "defaultValue": "[if(equals(parameters('solutionType'),'FhirServerCosmosDB'), false(), true())]",
```

Comment:
I think the intended behavior was to have no default value when sql to force a selection.

- The above is only true when the datalayer targets SQL.
- The value `true` is know to have problems when running on a webfarm (should be dev/test only for now)